backend/tests/unit/apps/slack/events/reaction_added/bot_feedback_test.py

```
@@ -0,0 +1,324 @@
+"""Tests for BotInteraction model, reaction feedback handler, and message_auto_reply hook."""
```

Ruff (0.15.18)

[warning] File `backend/tests/unit/apps/slack/events/reaction_added/bot_feedback_test.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)